### PR TITLE
Incorrect link in Vessel Viewer Data & Terminology page

### PR DESCRIPTION
**Pull request by fertandil87**
https://globalfishingwatch.atlassian.net/browse/VV-435

### CODE REVIEW

File: apps/vessel-history/features/terminology/__snapshots__/terminology-fishing-events.spec.tsx.snap [hunk 1]
```diff
@@ -27,7 +27,14 @@ exports[`TerminologyFishingEvents should render successfully 1`] = `
...
         https://globalfishingwatch.org/datasets-and-code/
       </a>
-      .
+      and FAQs:
+      <a
+        href="events.fishingEventsFaqsLink"
+        rel="noopener noreferrer"
+        target="_blank"
+      >
+        https://drive.google.com/file/d/1N4YRJ_yxAObEIWbaYM8l-YDPHS0bKe2N/view?usp=sharing/
```
Comment (rdgfuentes) [inline apps/vessel-history/features/terminology/__snapshots__/terminology-fishing-events.spec.tsx.snap:36]: Is this the new link for the updated FAQs? Leah said she had to update the doc in drive and that's why she blocked ths story.

**Review by rdgfuentes — APPROVED** [review]

File: apps/vessel-history/features/terminology/__snapshots__/terminology-fishing-events.spec.tsx.snap (hunk 1 above)
Comment (fertandil87) [reply apps/vessel-history/features/terminology/__snapshots__/terminology-fishing-events.spec.tsx.snap:36]: no @rdgfuentes , but now she can modify it in the translation

Comment (rdgfuentes) [reply apps/vessel-history/features/terminology/__snapshots__/terminology-fishing-events.spec.tsx.snap:36]: Perfect

**Review by rdgfuentes — CHANGES_REQUESTED** [review]
This PR should be merged into the `release/insurance-pilot` branch I think, right @fertandil87 ?

**Pull request merged**